paper-dropdown-menu.html

```diff
@@ -242,6 +242,16 @@
           verticalAlign: {
             type: String,
             value: 'top'
+          },
+
+          /**
+           * If you're using PaperInputBehavior to implement your own paper-input-like
+           * element, bind this to the `<input is="iron-input">`'s `readonly` property.
+           */
+          readonly: {
+            type: Boolean,
+            value: false,
+            observer: '_readonlyChanged'
```

Comment:
I believe we can get rid of this observer, as it can be expensive (this is triggered for all the dropdown menus) and it actually solves a very edge case scenario where someone updates this property while the dropdown is opened.
WDYT?